CONTRIBUTOR_LADDER.md

```diff
@@ -0,0 +1,111 @@
+# Contributor Ladder Template
+
+This is a light contributor ladder template for based of the [CNCF contribution ladder template](https://github.com/cncf/project-template/blob/main/CONTRIBUTOR_LADDER.md) 
+
+
+* [Contributor Ladder](#contributor-ladder-template)
+    * [Community Participant](#community-participant)
+    * [Contributor](#contributor)
+    * [Organization Member](#organization-member)
```

Comment:
I don't see descriptions of Org Member and Reviewer below. Are those defined?

Reply:
Given that Ratify is not an ORG maybe when we move it I can add it back.